### PR TITLE
feat : add note functionality

### PR DESCRIPTION
**Pull request by omkarpatke**
# Added add note functionality 

**Checks:**
- notePage.js
- noteContext.js
- api-calls.js

### CODE REVIEW

**Review by pradipgharge — APPROVED** [review]
Code looks clean and good to merge !

File: src/api-calls/api-calls.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const encodedToken = localStorage.getItem('token');
+
+    const addNoteToBackend = async(note) => {
+        try {
+            const response = await axios.post('/api/notes' , {note} ,{
+                headers: {
+                    authorization: encodedToken
+                }
+            })
+            console.log(response)
```
Comment (Sahil25061999) [inline src/api-calls/api-calls.js:12]: remove console.log if not needed.

File: src/context/note-context.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            try {
+                const response = await axios.get(`/api/notes` , 
+                {
+                    headers: {
+                        authorization: encodedToken
+                    }
+                })
+                setNotes(response.data.notes);
+            } catch (err) {
+                console.log(err);
```
Comment (Sahil25061999) [inline src/context/note-context.js:45]: use console.error here

**Review by Sahil25061999 — APPROVED** [review]
Looks good proper coventions are used, good readable code, suggested some minor changes.

**Pull request merged**